pkg/virt-handler/migration.go

```diff
@@ -21,28 +21,27 @@ package virthandler
 
 import (
 	"fmt"
-	"os"
+	"net"
 
-	"gopkg.in/yaml.v2"
+	v1 "kubevirt.io/api/core/v1"
 )
 
-// FindMigrationIP looks for dedicated migration network migration0 using the downward API and, if found, sets migration IP to it
-func FindMigrationIP(networkStatusPath string, migrationIp string) (string, error) {
-	var networkStatus []NetworkStatus
-
-	dat, err := os.ReadFile(networkStatusPath)
+// FindMigrationIP looks for dedicated migration network migration0. If found, sets migration IP to it
+func FindMigrationIP(migrationIp string) (string, error) {
```

Comment:
With this change [NetworkStatus](https://github.com/kubevirt/kubevirt/blob/85e9173c308434a6e2a9099f89eb631782e67bde/pkg/virt-handler/vm.go#L719-L723) struct is not used anymore and can be dropped.